poleno/attachments/models.py

```diff
@@ -135,8 +135,8 @@ def datachecks(superficial, autofix):
     # This check is a bit slow. We skip it if running from cron or the user asked for superficial
     # tests only.
     if superficial:
-        return
+        return []
```

Comment:
Vzdy si treba pretestovat vsetko, co implementujes/menis, ci to naozaj funguje. Aj `superficial` moznost. Pustaju sa pomocou `env/bin/python manage.py datacheck --superficial`.

Keby si si to otestoval, tak by si zistil, ze kedze funkcia `datachecks` uz v sebe nema `yield`, tak nie je generator. Takze `return` neukonci generator, ale vrati `None`. Co nie je iterator. Takze datachecks padnu.